app/assets/stylesheets/components/_bubble.scss

```diff
@@ -7,5 +7,25 @@
   font-weight: bold;
   margin: 10px 10px 10px 0px;
   padding: 15px 25px;
+  text-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25);
+}
+
+.invert-bubble {
+  background-color: white;
+  border-radius: 50px;
+  color: $blue;
+  border: 1px solid $blue;
+  display: inline-block;
+  font-size: 15px;
+  font-weight: bold;
+  margin: 10px 10px 10px 0px;
+  padding: 15px 25px;
   text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
```

Comment:
I would suggest removing this line for the invert-bubble. The text seems a little bit more clear when we don't have the shadow

![image](https://user-images.githubusercontent.com/20430030/205439391-a7eb360e-cf16-48f6-b576-55f9cb8a6fc7.png)

Reply:
okay, done